src/basic/env-util.c

```diff
@@ -776,6 +776,18 @@ int getenv_bool_secure(const char *p) {
         return parse_boolean(e);
 }
 
+int getenv_llu_secure(const char *p, unsigned base, unsigned long long *ret) {
```

Comment:
llu? That is such a weird type, given it's undefined length. I am pretty sure uint64_t is really the better choice here... i.e. getenv_uint64_secure()...

Reply:
Thanks I changed the function name and the argument type to `uint64_t *`.